src/main/java/gregtech/integration/jei/GTJeiPlugin.java

```diff
@@ -169,12 +169,11 @@ public void register(IModRegistry registry) {
                 MetaTileEntities.THERMAL_CENTRIFUGE,
                 MetaTileEntities.CHEMICAL_BATH,
                 MetaTileEntities.ELECTROMAGNETIC_SEPARATOR,
-                MetaTileEntities.SIFTER,
-                MetaTileEntities.ORE_WASHER
+                MetaTileEntities.SIFTER
         };
         for (MetaTileEntity[] machine : machineLists) {
-            if (machine.length < 1 || machine[0] == null) continue;
-            registry.addRecipeCatalyst(machine[0].getStackForm(), oreByProductId);
+            if (machine.length < GTValues.LV + 1 || machine[GTValues.LV] == null) continue;
```

Comment:
Why this change? It goes from excluding zero length arrays, to 1 length arrays. Even though none of these machines should be 1 length arrays, what if there is some specialty machine added that we want to add to this page in the future.

Reply:
The commit I linked above changed the indeces in the machine arrays to being adressable by `GTValues.<tier>`, effectively shifting all the indeces up by one, so this change is to make it get all the LV machines instead of trying to get ULV machines